instrumentation/clickhouse-client-0.5/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/clickhouse/ClickHouseClientTest.java

```diff
@@ -111,6 +116,15 @@ void testConnectionStringWithoutDatabaseSpecifiedStillGeneratesSpans()
                         .hasNoParent()
                         .hasAttributesSatisfyingExactly(
                             attributeAssertions("select * from " + tableName, "SELECT"))));
+
+    assertDurationMetric(
```

Comment:
added this assertion to get the test runner to pick up the metrics emitted by these tests. There might be another way I can hook these up that doesn't rely on a `waitAndAssert...` method